HOWTOSTART.md

```diff
@@ -0,0 +1,334 @@
+    "RouteToCustomAssetsFolder": "/customAssets",
+    "IdentityGuiLogoUrl": "/customAssets/avatarTest.png",
+    "Html5ClientLogoUrl": "/customAssets/avatarTest.png",
+    "FaviconLogoUrl": "/customAssets/faviconTest.png"
+  },
+  "IdentityServerConfig": {
+    "CookieSlidingExpiration": true,
+    "PathToJwtCertificate": "serverCore.pfx",
+    "PasswordForJwtCertificate": "<PasswordYouEnteredWhenCreatingTheCertificate>",
+    "WebClient": {
+      "RedirectUris": [
+        "https://localhost:44357/#origamClientCallback/",
+        "https://localhost:44357/#origamClientCallbackRenew/",
+        "https://localhost:5173/#origamClientCallback/",
+        "https://localhost:5173/#origamClientCallbackRenew/"
+      ],
+      "PostLogoutRedirectUris": [
+        "https://localhost:44357",
+        "https://localhost:5173"
+      ]
+    },
+    "ServerClient": {
+      "ClientSecret": "serverSecret"
+    }
+  },
+  "UserLockoutConfig": {
+    "MaxFailedAccessAttempts": 3
+  },
+  "PasswordConfig": {
+    "RequireDigit": "false",
+    "RequiredLength": "6",
+    "RequireNonAlphanumeric": "false",
+    "RequireUppercase": "false",
+    "RequireLowercase": "true"
+  },
+  "MailConfig": {
+    "Port": 587,
+    "UseSsl": "true",
+    "PickupDirectoryLocation": "<AbsolutePathToFolderWhereTheEmailsWillBeStored>"
+  },
+  "UserApiOptions": {
+    "RestrictedRoutes": [
+      "/api/private"
+    ],
+    "PublicRoutes": [
+      "/api/attachment",
+      "/api/public"
+    ]
+  },
+  "ClientFilteringConfig": {
+    "CaseSensitive": "false",
+    "AccentSensitive": "false"
+  },
+  "LanguageConfig": {
+    "Default": "en-US",
+    "Allowed": [
+      {
+        "Culture": "en-US",
+        "Caption": "English",
+        "ResetPasswordMailSubject": "Reset Password",
+        "ResetPasswordMailBodyFileName": "testResetPwFile.txt"
+      }
+    ]
+  }
+}
+```
+Make sure you set 
+- **PathToClientApp** to the absolute path to the
+`frontend-html\dist` folder where we built the frontend application before.
+- **PathToCustomAssetsFolder** to absolute path to a folder where you keep the application images. The images don't have to be there for development.
+- **PickupDirectoryLocation** to absolute path to a folder where the emails will be saved instead 
+of sending them to the actual email addresses. This is quite useful for development.
+- **PasswordForJwtCertificate** password you entered when creating the `serverCore.pfx` 
+file
+
+All back slashes in the paths should be escaped i.e. \\ instead of \.
+
+When you opened the solution Visual Studio created the file 
+```
+backend\Origam.Server\Properties\launchSettings.json
+```
+Open the file and set the **https** port to **44357**. You can ignore the http port number.
+
+
+
+Now go to Visual Studio and run the Origam.Server project. Everything should work and 
+your default browser should open with the login screen. Enter the following:
+
+**Username: testUser**
+
+**Password: 9ECZJ83w4UNQPjrR**
```

Comment:
This is probably not a problem of this document but: Why do we need such a memorable password for the Demo project?